apps/src/applab/designMode.js

```diff
@@ -569,11 +572,14 @@ function makeDraggable (jqueryElements) {
 
         elm.css({
           top: newTop,
-          left: newLeft
+          left: newLeft,
         });
 
         designMode.renderDesignWorkspace(elm[0]);
-      }
+      },
+      start: function () {
+        highlightElement(elm[0]);
+      },
     }).css({
```

Comment:
I think we want highlightElement to happen whenever an element is selected, so maybe it should be called from within editElementProperties instead? In any case it seems to me that the outline should appear after I drag an element into the visualization area from the toolbox the same way it would if I drag it around within the visualization area. Let me know if you disagree.

Likewise it seems like it should work if I click an element to select it (but do not drag), though it looks like that might work already.